apps/volontulo/templates/common/header.html

```diff
@@ -25,6 +25,11 @@ <h1 class="navbar-header">
                             <li>
                                 <a href="{% url 'logged_user_profile' %}">Moje konto</a>
                             </li>
+                            {% if user.is_staff %}
+                            <li>
+				                <a href="{% url 'admin_panel' %}">Panel administracyjny</a>
```

Comment:
You can use ?ts=4 in url to treat tab as 4 spaces... but, here we have only 3 spaces so should be fixed.